### PR TITLE
Set up endpoints and seed postgres

### PR DESCRIPTION
**Pull request by suejungshin**

### CODE REVIEW

File: database/postgres/pg-auth.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const dev = {
```
Comment (joshuahlee) [inline database/postgres/pg-auth.js]: you should have this inside of your git ignore and create an example.pg-auth.js file to upload instead for best practices

File: database/postgres/pg-schema.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
+CREATE TABLE IF NOT EXISTS zips(
+  zip_code INTEGER NOT NULL,
+  property_tax_rate DECIMAL(5,3),
+  CONSTRAINT zip_code PRIMARY KEY (zip_code)
+);
+
+-- property id is not set to serial/auto-increment to preserve behavior of endpoints from 1 to 1,000,000
+CREATE TABLE IF NOT EXISTS properties(
+  property_id INTEGER NOT NULL,
```
Comment (joshuahlee) [inline database/postgres/pg-schema.sql]: should this be SERIAL as well?

File: database/postgres/pg-auth.js (hunk 1 above)
Comment (suejungshin) [reply database/postgres/pg-auth.js]: good idea, thanks!

File: database/postgres/pg-schema.sql (hunk 2 above)
Comment (suejungshin) [reply database/postgres/pg-schema.sql]: For now I am having the for loop create the index so I can control it to be between 1 and 1,000,000 and know what endpoint websites to go to vs. if postgres assigns the index and starts at some other number. But for the final product, may consider switching it over to having postgres assign it. Thanks!

**Pull request merged**